rest/data/project_test.go

```diff
@@ -420,36 +457,53 @@ func TestUpdateProjectVarsByValue(t *testing.T) {
 }
 
 func TestUpdateProjectVarsByValueWithEnabledOnly(t *testing.T) {
```

Comment:
Do we need to test for hidden/deleted projects?

Reply:
Yup, that's tested [here](https://github.com/evergreen-ci/evergreen/blob/c842e8a1bdf79ab03f4f8fc775190500ed120bfd/rest/data/project_test.go#L773).